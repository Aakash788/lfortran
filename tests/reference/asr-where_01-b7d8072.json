{
    "basename": "asr-where_01-b7d8072",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/where_01.f90",
    "infile_hash": "761e682dad0855b945982b56d551b9f6a7aacba6ee8c0b4586926ad1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-where_01-b7d8072.stdout",
    "stdout_hash": "50e830f430599fb348a2b00de3f04fed23122909553c674375db05b6",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}