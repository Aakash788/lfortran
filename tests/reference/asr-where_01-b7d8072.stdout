(TranslationUnit
    (SymbolTable
        1
        {
            where_01:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [((IntegerConstant 1 (Integer 4 []))
                                    (IntegerConstant 10 (Integer 4 [])))])
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            b:
                                (Variable
                                    2
                                    b
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4 [((IntegerConstant 1 (Integer 4 []))
                                    (IntegerConstant 10 (Integer 4 [])))])
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    where_01
                    []
                    [(Where
                        (IntegerCompare
                            (Var 2 a)
                            GtE
                            (IntegerConstant 0 (Integer 4 []))
                            (Logical 4 [((IntegerConstant 1 (Integer 4 []))
                            (IntegerConstant 10 (Integer 4 [])))])
                            ()
                        )
                        [(=
                            (Var 2 b)
                            (IntegerConstant 1 (Integer 4 []))
                            ()
                        )]
                        [(=
                            (Var 2 b)
                            (IntegerConstant 0 (Integer 4 []))
                            ()
                        )]
                    )
                    (Where
                        (IntegerCompare
                            (Var 2 a)
                            GtE
                            (IntegerConstant 0 (Integer 4 []))
                            (Logical 4 [((IntegerConstant 1 (Integer 4 []))
                            (IntegerConstant 10 (Integer 4 [])))])
                            ()
                        )
                        [(=
                            (Var 2 b)
                            (IntegerConstant 1 (Integer 4 []))
                            ()
                        )]
                        [(=
                            (Var 2 b)
                            (IntegerConstant 0 (Integer 4 []))
                            ()
                        )]
                    )
                    (Where
                        (IntegerCompare
                            (Var 2 a)
                            GtE
                            (IntegerConstant 0 (Integer 4 []))
                            (Logical 4 [((IntegerConstant 1 (Integer 4 []))
                            (IntegerConstant 10 (Integer 4 [])))])
                            ()
                        )
                        [(=
                            (Var 2 b)
                            (IntegerConstant 1 (Integer 4 []))
                            ()
                        )]
                        [(=
                            (Var 2 b)
                            (IntegerConstant 0 (Integer 4 []))
                            ()
                        )]
                    )]
                )
        })
    []
)
